conda/recipes/libcuvs/recipe.yaml

```diff
@@ -0,0 +1,408 @@
+            - libcusparse-dev =${{ cuda11_libcusparse_host_version }}
+          else:
+            - cuda-cudart-dev
+            - cuda-profiler-api
+            - libcublas-dev
+            - libcurand-dev
+            - libcusolver-dev
+            - libcusparse-dev
+      run:
+        - ${{ pin_subpackage("libcuvs", exact=True) }}
+        - ${{ pin_compatible("cuda-version", upper_bound="x", lower_bound="x") }}
+        - libraft-headers =${{ minor_version }}
+        - nccl
+        - if: cuda_major == "11"
+          then:
+            - cuda-cudart
+            - libcublas
+            - libcurand
+            - libcusolver
+            - libcusparse
+      ignore_run_exports:
+        by_name:
+          - cuda-cudart
+          - cuda-version
+          - libaio
+          - libboost
+          - libcublas
+          - libcurand
+          - libcusolver
+          - libcusparse
+          - librmm
+          - mkl
+          - nccl
+          - if: cuda_major == "11"
+            then: cudatoolkit
+    about:
+      homepage: ${{ load_from_file("python/libcuvs/pyproject.toml").project.urls.Homepage }}
+      license: ${{ load_from_file("python/libcuvs/pyproject.toml").project.license.text }}
+      summary: libcuvs test executables
+
+  - package:
+      name: libcuvs-examples
+      version: ${{ version }}
+    build:
+      string: cuda${{ cuda_major }}_${{ date_string }}_${{ head_rev }}
+      dynamic_linking:
+        overlinking_behavior: "error"
+      script:
+        content: |
+          # Remove `-fdebug-prefix-map` line from CFLAGS and CXXFLAGS so the
+          # incrementing version number in the compile line doesn't break the
+          # cache
+          set -x
+          export CFLAGS=$(echo $CFLAGS | sed -E 's@\-fdebug\-prefix\-map[^ ]*@@g')
+          export CXXFLAGS=$(echo $CXXFLAGS | sed -E 's@\-fdebug\-prefix\-map[^ ]*@@g')
+          set +x
+
+          ./build.sh examples
+
+        secrets:
+          - AWS_ACCESS_KEY_ID
+          - AWS_SECRET_ACCESS_KEY
+          - AWS_SESSION_TOKEN
+        env:
+          CMAKE_C_COMPILER_LAUNCHER: ${{ env.get("CMAKE_C_COMPILER_LAUNCHER") }}
+          CMAKE_CUDA_COMPILER_LAUNCHER: ${{ env.get("CMAKE_CUDA_COMPILER_LAUNCHER") }}
+          CMAKE_CXX_COMPILER_LAUNCHER: ${{ env.get("CMAKE_CXX_COMPILER_LAUNCHER") }}
+          CMAKE_GENERATOR: ${{ env.get("CMAKE_GENERATOR") }}
+          PARALLEL_LEVEL: ${{ env.get("PARALLEL_LEVEL") }}
+          RAPIDS_ARTIFACTS_DIR: ${{ env.get("RAPIDS_ARTIFACTS_DIR") }}
+          SCCACHE_BUCKET: ${{ env.get("SCCACHE_BUCKET") }}
+          SCCACHE_IDLE_TIMEOUT: ${{ env.get("SCCACHE_IDLE_TIMEOUT") }}
+          SCCACHE_REGION: ${{ env.get("SCCACHE_REGION") }}
+          SCCACHE_S3_USE_SSL: ${{ env.get("SCCACHE_S3_USE_SSL") }}
+          SCCACHE_S3_NO_CREDENTIALS: ${{ env.get("SCCACHE_S3_NO_CREDENTIALS") }}
+          SCCACHE_S3_KEY_PREFIX: libcuvs-${{ env.get("RAPIDS_CONDA_ARCH") }}
```

Comment:
Probably a discussion for later -- we key our sccache on `${library}-${arch}`. In that cache, we have all kinds of artifacts that are incompatible because they are produced for different CUDA versions. Should CUDA version be a part of the sccache key? Or should we remove the arch prefix? The status quo isn't making a lot of sense as I consider it further.

Reply:
hm, that's a good point.  I would defer to some of the more experienced `sccache` folk but it does seem like we would benefit from a more specific key